config/admins.txt

```diff
@@ -135,3 +135,4 @@ Naksuasdf = Game Master
 MrDoomBringer = Game Master
 shizcalev = Game Master
 NicBR = Game Master
+loserwastaken = Game Master

```

Comment:
Hmmmmm...

Reply:
that was for testing and it doesn't matter because the server uses SQL-based badmins